### PR TITLE
Add CLI options for loading events and labels from JSON

### PR DESCRIPTION
**Pull request by osteele**
Also move the sample data to data files.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,37 +98,37 @@ OK
 ### abe.olin.build/events/
 
 | HTTP Method | Action              |
-| ----------- | ------------------- |
```
Comment (osteele) [inline README.md]: My code format editor plugin made these changes. If someone else's formatter wants them the other way I can teach it not to so we don't get into dueling commits.

File: abe/sample_data.py [hunk 2]
```diff
@@ -1,14 +1,23 @@
 #!/usr/bin/env python
-# -*- coding: utf-8 -*-
```
Comment (osteele) [inline abe/sample_data.py]: Not necessary for Python 3.  Removing as I otherwise edit files.

File: abe/sample_data.py [hunk 3]
```diff
@@ -230,7 +88,6 @@ def load_data(
         for event in event_data:
             for key, value in event.items():
                 if type(value) is datetime:  # convert to UTC from EST
-                    og = value
```
Comment (osteele) [inline abe/sample_data.py]: Who's og, anyway!? (Was causing a linter warning.)

File: abe/sample_data.py [hunk 4]
```diff
@@ -252,7 +109,43 @@ def load_data(
         for ics in ics_data:
             db.ICS(**ics).save()
 
+SampleData = namedtuple('SampleData', ['events', 'labels', 'icss'])
+
+
+def load_sample_data():
+    """Return hard-coded sample data. Event and label data are read from the test
+    data directory. This exists as a separate function so that its data can be
+    shared between the Heroku postdeploy script and the --use-defaults
```
Comment (newsch) [inline abe/sample_data.py]: Is `use-defaults` a command for `sample_data.py` or something else?

File: abe/sample_data.py (hunk 3 above)
Comment (newsch) [reply abe/sample_data.py]: I'm pretty sure that dangling initialization is my fault.

**Comment by newsch** [comment]
Thanks Oliver, this has been an unspoken "TODO" for a while now.
- Are these `.json` files modified exports from MongoDB, ABE, or somewhere else?
- Looks like `click` was already in `requirements.txt` - a Flask dependency I think.

**Comment by osteele** [comment]
* The JSON files were extracted from `sample_data.py`, with (1) the `date(…)` calls replaced by ISO date or date time strings, and (2) a couple of events had two “label” fields, which works in a Python dict literal but not in JSON.
* Right, Flask and its CLI use Click (and I think they’re by the same author), which is why I went with that one.

**Pull request merged**